feedback/static/html/feedback.html

```diff
@@ -0,0 +1,17 @@
+<div class="feedback_block">
+  <form action="" onsubmit="return false;">
+    <fieldset class="feedback_likert_field">
+      <legend class="feedback_likert_header">{likert_prompt}</legend>
+      <div class="feedback_likert_scale">{scale}</div>
+    </fieldset>
+
+    <div class="feedback_header_div">
+      <label class="feedback_header" for="feedback_freeform_textarea">{freeform_prompt}</label>
```

Comment:
@pmitros this label does not programmatically link to the textarea below.  the for attribute must take an IDref.  This references the name attribute. Recommend adding an IDref to the textarea below.  You may need to append a random value to the end of the ID since multiple Feedback blocks on one page would cause uniqueness errors that would actually impact accessibility (not all do).